docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
-Available threat indicator match data appears in the **Threat Match Detected** subsection.
+Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
+
+Click **Threat intelligence** to expand the **Threat intelligence** sub-tab in the left panel. Within the sub-tab, matched threats are organized into two sections: <<threat-match-detected-section>> and <<enriched-with-threat-intelligence-section>>. Matched threats are ordered in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
```

Comment:
super small nit, but aren't `ordered in reverse chronological order` and `with the most recent at the top` saying the same thing?

Reply:
It's saying similar things, but I think we should keep the second part because it adds clarity.